app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="spacing">20dp</dimen>
+    <dimen name="item_genres_h">100dp</dimen>
+    <dimen name="item_genres_w">100dp</dimen>
+    <dimen name="fab_margin">16dp</dimen>
+    <dimen name="item_album_h">70dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.item_album_h` appears to be unused
 Unused resources make applications larger and slow down builds.